com.unity.sg2/Editor/GraphDeltaRegistry/Registry/Registry.cs

```diff
@@ -85,9 +85,9 @@ public class Registry
         public Registry()
         {
             Register<ContextBuilder>();
-            Register<ReferenceNodeBuilder>();
             Register<PropertyContext>();
             defaultTopologies = new GraphHandler(this);
+            Register<ReferenceNodeBuilder>();
```

Comment:
since RegerenceNodeBuilder makes a function, it needs a default topo. so the graph has to be instantiated